benchmarking/commons/launch_remote_local.py

```diff
@@ -111,42 +118,114 @@ def launch_remote(
         command line arguments
     :param extra_args: Extra arguments for command line parser, optional
     """
-    args, method_names, seeds = parse_args(methods, extra_args)
-    experiment_tag = args.experiment_tag
-    suffix = random_string(4)
-    benchmark = get_benchmark(args, benchmark_definitions)
-    master_random_seed = get_master_random_seed(args.random_seed)
+    configuration = config_from_argparse(extra_args, LOCAL_BACKEND_EXTRA_PARAMETERS)
+    launch_remote_experiments(
+        configuration, entry_point, methods, benchmark_definitions
+    )
+
+
+def launch_remote_experiments(
+    configuration: ConfigDict,
+    entry_point: Path,
+    methods: MethodDefinitions,
+    benchmark_definitions: RealBenchmarkDefinitions,
+):
+    """
+    Launches sequence of SageMaker training jobs, each running an experiment
+    with the local backend. The loop runs over methods selected from ``methods``
+    and repetitions.
 
+    Combination of ``requirements.txt``: Dependencies for ``entry_point`` are the
+    union of Syne Tune dependencies and dependencies of the training script
+    (not contained in its SageMaker framework). For the former, we scan
+    ``entry_point.parent`` for a file named ``requirements*.txt``. If this is not
+    found, we create a default one called ``requirements-synetune.txt``. This is
+    then combined with the ``requirements.txt`` file for the training script
+    (if any), and the union is written to ``requirements.txt`` in
+    ``entry_point.parent``.
+    If you like to control the Syne Tune requirements (the default ones are
+    ``"extra"``, which can be a lot), place a file ``requirements_synetune.txt`` in
+    ``entry_point.parent``.
+
+
+    :param configuration: ConfigDict with parameters of the benchmark.
+            Must contain all parameters from
+            hpo_main_local.LOCAL_LOCAL_BENCHMARK_REQUIRED_PARAMETERS
+    :param entry_point: Script for running the experiment
+    :param methods: Dictionary with method constructors; one is selected from
+        command line arguments
+    :param benchmark_definitions: Definitions of benchmarks; one is selected from
+        command line arguments
+    """
+    configuration.check_if_all_paremeters_present(LOCAL_BACKEND_EXTRA_PARAMETERS)
+    configuration.expand_base_arguments(LOCAL_BACKEND_EXTRA_PARAMETERS)
+
+    method_names = (
+        [configuration.method]
+        if configuration.method is not None
+        else list(methods.keys())
+    )
+    benchmark = get_benchmark(configuration, benchmark_definitions)
+    master_random_seed = get_master_random_seed(configuration.random_seed)
     synetune_requirements_file = find_or_create_requirements_txt(
         entry_point, requirements_fname="requirements-synetune.txt"
     )
     combine_requirements_txt(synetune_requirements_file, benchmark.script)
+    extra_sagemaker_hyperparameters = {"verbose": int(configuration.verbose)}
```

Comment:
These are the only ones? Just check again whether nothing is dropped

Reply:
I think this maps bool to int before the parameter is passed to sagemaker; this was the logic before the refator. Do we not need it now?